sys-process/watchpid/watchpid-0.1-r2.ebuild

```diff
@@ -1,28 +1,21 @@
-# Copyright 1999-2016 Gentoo Foundation
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
 
-inherit multilib-minimal
+inherit multilib-minimal toolchain-funcs
 
 DESCRIPTION="Watches a process for termination"
-SRC_URI="mirror://gentoo/${PN}_${PV}.tar.gz"
 HOMEPAGE="http://www.codepark.org/"
-KEYWORDS="amd64 ppc x86"
+SRC_URI="mirror://gentoo/${PN}_${PV}.tar.gz"
 
-SLOT="0"
 LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="amd64 ppc x86"
 
-DOCS=( README AUTHORS )
+PATCHES=( "${FILESDIR}"/${P}-cflags.patch )
```

Comment:
I prefer patches on their own line to minimise diff/effort when adding more in future

Reply:
Sure, I usually do a one-liner if it's a single patch, but I can change it ;)

Reply:
I get it, the main reason I started to think about it other than effort on adding a new patch is that things like this make it _way_ easier to sync in e.g. the Prefix overlay (especially when they had weird 'cross-diffs').